ArmoniK.SDK.Client.Test/src/ArmoniK.SDK.Client.Test.cpp

```diff
@@ -172,13 +179,14 @@ TEST(testSDK, testAddInt) {
   // Create the task options
   ArmoniK::Sdk::Common::TaskOptions session_task_options(
       "libArmoniK.SDK.Worker.Test.so", config.get("WorkerLib__Version"), "End2EndTest", "AdditionService");
+  session_task_options.max_retries = 1;
 
   // Create the properties
   ArmoniK::Sdk::Common::Properties properties{config, session_task_options};
 
   // Create the logger
-  ArmoniK::Api::Common::logger::Logger logger{ArmoniK::Api::Common::logger::writer_console(),
-                                              ArmoniK::Api::Common::logger::formatter_plain(true)};
+  armonik::api::common::logger::Logger logger{armonik::api::common::logger::writer_console(),
+                                              armonik::api::common::logger::formatter_plain(true)};
```

Comment:
the grpc proto are generated under armonik::api::grpc
The api are generated under armonik::api

They are both "low-level" and thus being in the same namespace makes sense

For ArmoniK.Sdk, since it's a higher level Api, we'll use camel case